### ISSUE TITLE
Acrobat Screenreader gives empty page warning

### ISSUE THREAD

**Issue by aleph-71** [AndyClifton/accessibility#32]
I can compile the test documents no problem. However, when I try to get Acrobat Pro to read the document out loud I only get an empty page warning. Surely, making it impossible for the screen reader to read the document runs counter to the goal of making pdfs accessible. What's going on?

**Comment by AndyClifton** [comment]
Yes, an ability to read a PDF out loud would be one of the effects of the package if the PDF is properly structured. 

The error you're reporting means that there's an error somewhere in the document or package, and we'll have to figure out what that error is. 

Thanks for raising this issue!

**Comment by aleph-71** [comment]
Dear Andy, 

Thanks a lot. 

One of the things that I noticed while playing around with the 'tagpdf' package - where, of course, you enter the tags manually for the time being - was that this empty page error was raised by Acrobat Pro's screen reader whenever the document wasn't exhaustively tagged. Since I have not tried to understand how the accessibility package works, I can only conjecture that maybe the tagging is too fragmentary from acrobat's perspective? 

I'm happy to help in whatever way I can but I have, alas, no experience in TeX/LaTeX coding and no knowledge of pdf.

**Comment by viktoriasee** [comment]
I think this is just another symptom of #12 .
